File: aoj/itp1/2c/main.cpp
#include <iostream>
#include <utility>
#include <vector>
#include <string>
#include <sstream>
#include <algorithm>
#include <map>

using namespace std;

int main() {
  cin.tie(0);
  ios::sync_with_stdio(false);

  int a, b, c;
  cin >> a >> b >> c;
  int A = min(min(a, b), c);
  int C = max(max(a, b), c);
  int B = a + b + c - A - C;  
  cout << A << ' ' << B << ' ' << C << endl;

  return 0;
}

File: aoj/itp1/2c/main.ml
Scanf.scanf "%i %i %i\n" (fun a b c ->
  let aa = min a (min b c) in
  let cc = max a (max b c) in
  let bb = a + b + c - aa - cc in
  Printf.printf "%d %d %d\n" aa bb cc
)